pennylane/resource/first_quantization.py

```diff
@@ -144,6 +144,22 @@ def _flatten(self):
             ("vectors", self.vectors),
         )
 
+    # pylint: disable=arguments-differ
+    @classmethod
+    def _primitive_bind_call(
+        cls,
+        n,
+        eta,
+        omega=None,
+        error=0.0016,
+        charge=0,
+        br=7,
+        vectors=None,
+    ):
+        return cls._primitive.bind(
+            n, eta, omega=omega, error=error, charge=charge, br=br, vectors=vectors
+        )
```

Comment:
I'm torn on this one: I read through it, trying to find out whether there is custom handling of any (kw)args. The number of arguments makes it hard to see this at once, so maybe the following is actually "more readable" in the end?
```suggestion
    # pylint: disable=arguments-differ
    @classmethod
    def _primitive_bind_call(cls, n, eta, **kwargs):
        return cls._primitive.bind(n, eta, **kwargs)
```